docker-mac.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+
+def=$'\e[39m'
+yel=$'\e[93m'
+blue=$'\e[34m'
+green=$'\e[32m'
+cyan=$'\e[96m'
+
+# Install Brew  
+echo $blue"==> brew checking"
+brew_version="$(brew --version 2>/dev/null)"
+echo $def"${brew_version}"
+if [[ "$brew_version" == *"Homebrew"* ]]; then
+  echo $green"brew is installed"
+else
+  echo $yel"brew does not exist"
+  echo $def"Please read https://brew.sh/ for information about brew"
+  echo $cyan"Installing homebrew"
+  /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+fi
+
+# Install Docker-Desktop on Mac
+docker_info="$(docker info 2>/dev/null | grep ' Operating System: Docker Desktop')"
+echo $def"${docker_info}"
+if [[ "$docker_info" == *"Operating System: Docker Desktop"* ]]; then
+  echo $green"Docker Desktop is installed"
+else
+  echo $cyan"Installing Docker Desktop on Mac"$def
+  brew cask install Docker
+  echo $cyan"You are prompted to authorize Docker.app with your system password"$def
+  sleep 2
+  open /Applications/Docker.app
+  until docker info 2>/dev/null | grep "Server Version:";
+  do
+  echo $yel"Waiting for Docker to start. Please ensure to grant privileged access"$def;
+  sleep 4; 
+  done && echo "Docker Desktop is running"
+  until kubectl version --short 2>/dev/null | grep "Server Version: v";
+  do
+  echo $cyan"Please enable kubernetes in preferences menu and Waiting for Kubernetes to start"$def;
```

Comment:
```suggestion
  echo $cyan"Waiting for Kubernetes to start. Please enable Kubernetes in preferences menu"$def;
```

In ordinary sentences, capitalise first letters of sentences and proper nouns (names of people, places, things) only. 

I know English is an incoherent and inconsistent language and is even tougher written, so I apologise in advance for correcting you on it.
The problem is our target audience are majority native English speakers and it looks unprofessional if we don't get it right. Again, apologies.

Reply:
updated